### ISSUE TITLE
Us holiday calendar has Easter on the 4th April.

### ISSUE THREAD

**Issue by MadMattAu** [thundernest/thunderbird-website#297]
The subject says it all,  the Easter date is way off.  It was reported in support https://support.mozilla.org/en-US/questions/1374202 but on downloading and looking it is not missing,  just on the wrong date. It is specified as the 4th April.  The earliest date on which Easter can arrive.  The other issue is it appears to mark the Friday,  but Easter is based on a Sunday date.

Full moon Saturday 16th and Easter is Sunday 17th this year.

**Comment by Sancus** [comment]
I think I'm going to drop the crowdsourced calendars in favor of an automated approach using https://github.com/dr-prodigy/python-holidays or possibly https://calendarific.com/

They are just way too prone to going out of date or on errors and it's a very poor use of developer time to manually correct .ics files or review calendar PRs.

**Comment by MadMattAu** [comment]
> I think I'm going to drop the crowdsourced calendars in favor of an automated approach using https://github.com/dr-prodigy/python-holidays or possibly https://calendarific.com/
> 
> They are just way too prone to going out of date or on errors and it's a very poor use of developer time to manually correct .ics files or review calendar PRs.

I agree, with dropping the crowdsourcing.  No pushback from this quarter.

**Comment by mathgirl25** [comment]
Same with Election Day.  The calendar shows it as Tuesday, November 1, 2022.  Election Day is actually Tuesday, November 8, 2022.
